README.md

```diff
@@ -1,20 +1,32 @@
 video-mode-plugin
 =================
 
-A BrightAuthor plugin that changes the video resolution, frame rate, color space and color depth.
+A BrightSign plugin that changes the video resolution, frame rate, color space and color depth.
 
-# Usage
+SetSingleScreenVideoMode.brs for non-series 5 single screen players.
+SetMultiScreensVideoMode.brs for series 5 multi screens players.
 
```

Comment:
Add line item mentioning what videoModePlugin.brs is similarly to how the single and multiscreen plugins are documented.

Reply:
Added one-line item mentioning what videoMode_plugin.brs is.